### ISSUE TITLE
Crash when closing the app

### ISSUE THREAD

**Issue by blackhamm3rjack** [serge-rgb/milton#79]
As said in the title, the application always crashes when I close the app, be it with Ctrl+Q, manual click on the window's X button or Alt+F4. I started with a fresh install and the only modification I did to configuration settings was the background color. The version is 1.5.1 and the log file says the following:
```
CommandLine is 
Running Milton
Initializing SDL... Done.
Loading preferences... Could not open file for writing prefs
Done.
Creating Milton Window
Created OpenGL context with version 4.6.0 NVIDIA 390.77
    and GLSL 4.60 NVIDIA
Easy Tab Failed to load. Code -3Maximum viewport dimensions, 32768.000000x32768.000000
Vendor string: "NVIDIA Corporation"
Milton: loading file
Increased guid to 1
```

It's not really a big issue but if it's fixed, it would be nice.

**Comment by blackhamm3rjack** [comment]
While debugging with Visual Studio, I get the error 0xC0000005: "Access violation reading location 0x0000000000000630". It looks like a faulty pointer.

**Comment by serge-rgb** [comment]
This is already fixed in master (issue #64), I just haven't done a release with the fix
